parallelization_test/parallelization-benchmark.ipynb

```diff
@@ -0,0 +1,1325 @@
+{
```

Comment:
<p>How about adding a brief title and description in a markdown cell above, sth like:</p><h1>Parallization analysis</h1><p>This notebook analyses the time required for running <code>get-sequences</code> with different number of jobs (1 and 4) in 3 iterations.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bokulich-lab/q2-fondue-examples/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>